### PR TITLE
chore: add context to capture widget

### PR DESCRIPTION
**Pull request by evillegas01**

### CODE REVIEW

**Comment by dayures** [comment]
@simonadomnisoru who can help with this PR?

File: docs/user/using-the-capture-app.md [hunk 1]
```diff
@@ -998,7 +998,7 @@ You can complete the enrollment by clicking the complete button. When there are
 
 ![](resources/images/enrollment-widget-comment.png)
 
-The enrollment comment widget displays comments and allows addition of comments, associated with the current enrollment. 
+The enrollment comment widget (equivalent of the Notes widget in Tracker Capture app) displays comments and allows addition of comments, associated with the current enrollment.
```
Comment (simonadomnisoru) [inline docs/user/using-the-capture-app.md:1001]: Hi @evillegas01 
There was some work done to change the word `comment` for `note` throughout the whole app, including this line of [docs](https://github.com/dhis2/capture-app/pull/3549/files#diff-1a5919ed1ac425bce5fe1126ef801d781f2d0288088bcba3bdc61a53128096abR999). Is your change still relevant or the refactor is understandable enough from your point of view? 
IMHO, we should avoid mentioning the old Tracker Capture in the Capture docs. I think these kinds of feature parity matches belong more in some kind of migration steps instructions or the release notes. WDYT?
Thanks!

Comment (evillegas01) [reply docs/user/using-the-capture-app.md:1001]: Thank you for your feedback, Simona. All your changes are sufficiently explanatory, so from my point of view this pr is not needed anymore.
Thanks a lot.

**Comment by simonadomnisoru** [comment]
The docs changes made in https://github.com/dhis2/capture-app/pull/3549 are sufficiently explanatory. Thanks for your contribution @evillegas01!

**Pull request closed without merging**